internal/provider/resources/webhooks.go

```diff
@@ -34,7 +33,7 @@ type WebhookResourceModel struct {
 	Name             types.String               `tfsdk:"name"`
 	Description      types.String               `tfsdk:"description"`
 	Enabled          types.Bool                 `tfsdk:"enabled"`
-	Template         jsontypes.Normalized       `tfsdk:"template"`
+	Template         types.String               `tfsdk:"template"`
```

Comment:
This better matches the API spec where the field type is `string`: https://app.prefect.cloud/api/docs#tag/Webhooks